src/pages/components/range.mdx

```diff
@@ -1,10 +1,106 @@
 ---
 title: Range
 description: >
-  Description will be here
+  A Range is an input that lets users select a value from within a given range.
+package: '@zendeskgarden/react-forms'
+components:
+  - forms/src/elements/common/Field.tsx
+  - forms/src/elements/common/Hint.tsx
+  - forms/src/elements/common/Label.tsx
+  - forms/src/elements/common/Message.tsx
+  - forms/src/elements/Range.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To choose a value in a given range of minimum and maximum values</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To choose a range of numbers between two values, use a
```

Comment:
- If the choices aren’t in a continuous range, use a Menu instead
- To compare between several options, use a Radio instead
- To define a range of values, use a Multi-thumb Range instead